### PR TITLE
Fix sidebar navigation links and errors in md files

### PR DESCRIPTION
**Pull request by ranatrk**
#### What this PR resolves:
-Navigating to other wiki links don't work because the current namespace is appended to the previous when working with the different wiki repos:
eg: /wiki/foundation/grid instead of /wiki/grid
-Docs added from docsites include example folders that dont have _sidebar.md or readme.md files resulting in errors

#### Changes proposed in this PR:

- The whole url is changed to navigate from one wiki to the other instead of appending to the current url 
-Add temporary _sidebar.md and readme.md files

**Fixes**: #26

### CODE REVIEW

File: packages/dm_base/blueprints/wiki/routes.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 from blueprints.wiki import blueprint
 from flask import render_template, send_file, request
-from flask import abort, redirect, url_for
+from flask import abort, redirect, url_for, send_from_directory
```
Comment (rkhamis) [inline packages/dm_base/blueprints/wiki/routes.py]: please remove unused import

File: packages/dm_base/blueprints/wiki/routes.py [hunk 2]
```diff
@@ -19,11 +19,11 @@ def index():
 @blueprint.route('')
 def index_sub(sub):
     return render_template('index_docsify.html')
-
+
```
Comment (rkhamis) [inline packages/dm_base/blueprints/wiki/routes.py]: please clean whitespace. I suggest you use a linter to show you these errors, will make your life a lot easier

**Review by rkhamis — CHANGES_REQUESTED** [review]

File: packages/dm_base/blueprints/wiki/routes.py (hunk 1 above)
Comment (ranatrk) [reply packages/dm_base/blueprints/wiki/routes.py]: I updated the commit.

File: packages/dm_base/blueprints/wiki/routes.py (hunk 2 above)
Comment (ranatrk) [reply packages/dm_base/blueprints/wiki/routes.py]: I updated it for now but will check out a linter for the future. Thank you.

**Review by rkhamis — APPROVED** [review]

**Pull request merged**